### PR TITLE
CleanProFinder.Server: get users requests endpoints

### PR DESCRIPTION
**Pull request by s1uns**

### CODE REVIEW

File: CleanProFinder/CleanProFinder.Server/Controllers/RequestController.cs [hunk 1]
```diff
@@ -53,5 +53,34 @@ public async Task<IActionResult> GetActiveRequests(CancellationToken cancellatio
             var result = await _mediator.Send(new GetActiveRequestsQuery(), cancellationToken);
             return ConvertFromServiceResponse(result);
         }
+
+
+        /// <summary>
+        /// Get request by id.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [HttpGet("my-active-requests/{id}")]
```
Comment (Wednumi) [inline CleanProFinder/CleanProFinder.Server/Controllers/RequestController.cs:62]: please remove "active" from urls, user can view all his requests

File: CleanProFinder/CleanProFinder.Server/Features/Requests/GetRequestByIdQuery.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+            private async Task<ServiceResponse<RequestDto>> UnsafeHandleAsync(GetRequestByIdQuery request,
+                CancellationToken cancellationToken)
+            {
+                var serviceRequest = await _context
+                    .Requests
+                    .Include(r => r.Premise)
+                    .Include(r => r.Services)
+                    .FirstOrDefaultAsync(r => r.Id == request.Id)
+;
```
Comment (Wednumi) [inline CleanProFinder/CleanProFinder.Server/Features/Requests/GetRequestByIdQuery.cs:54]: add check that user who try to get this info is request owner

File: CleanProFinder/CleanProFinder.Server/Controllers/RequestController.cs (hunk 1 above)
Comment (s1uns) [reply CleanProFinder/CleanProFinder.Server/Controllers/RequestController.cs:62]: fixed

File: CleanProFinder/CleanProFinder.Server/Features/Requests/GetRequestByIdQuery.cs (hunk 2 above)
Comment (s1uns) [reply CleanProFinder/CleanProFinder.Server/Features/Requests/GetRequestByIdQuery.cs:54]: fixed

**Comment by firtap** [comment]
Please add PremiseId to RequestDto for a user to have the option to view all information about the premise in request.

**Review by firtap — APPROVED** [review]

**Review by Wednumi — APPROVED** [review]

**Pull request merged**